commands.pm

```diff
@@ -20,8 +20,7 @@ BEGIN {
     $ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
 }
 
-# Automatically enables "strict", "warnings", "utf8" and Perl 5.10 features
-use Mojolicious::Lite;
+use Mojolicious::Lite -signatures;
```

Comment:
why do we need to enable signatures twice?

Reply:
I don't know. Do you think we should just remove this line?

Reply:
If I remove the import then t/07-commands.t fails on unknown definition of "app". If I remove the `-signatures` flag then we get "The signatures feature is experimental"

Reply:
Yes, we're using `Mojolicious::Lite` here.

> If I remove the -signatures flag then we get "The signatures feature is experimental"

Maybe that module is special then. So I guess we can just keep the `-signatures` then.

Reply:
hm, ok. I guess it doesn't hurt to have it twice